enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -87,50 +87,208 @@ Out of scope for this feature
 
 ## Proposal
 
-The Selective Policy Enforcement feature adds an API to the policy
-framework to allow enforcement of inform policies to selected
-clusters. The effect of this would be the equivalent of setting the
-complianceType of child (cluster namespace copy) policies to enforce.
+The Selective Policy Enforcement feature extends the policy APIs to
+allow enforcement of inform policies to selected clusters. The effect
+of this would be the equivalent of setting the remediationAction of
+child (cluster namespace copy) policies to enforce for those selected
+clusters.
 
-The proposed API would be the addition of a remediationAction of
-“controlled” to Policy. In this mode the policy would act as an inform
-policy until the user, controller, or higher level orchestration
-enables remediation directly on the child policy.
+There are two proposed API changes to allow for selective policy
+enforcement shown in this sample:
 
-The mechanism to enable enforcement of a child policy would be to add
-an annotation to the child policy which enables enforcement
+```
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+...
+remediationActionOverride:
+    remediationAction: [null|enforce]
+    subFilter: [fale|true]
```

Comment:
Does this field belong to the binding itself? I thought this was for the binding part, not so much for the action override?